src/css/neo4j-docs.css

```diff
@@ -184,21 +179,16 @@ div.roles > span.role::after {
   margin-bottom: 0;
 }
 
-body.deprecated article h2::after,
-body.deprecated article h3::after,
-body.deprecated article h4::after,
-div.deprecated h2::after,
-div.deprecated h3::after,
-div.deprecated h4::after,
+body.deprecated article h1::after,
 .deprecated dt::after,
 p.deprecated::before,
 section.deprecated .title::after,
 .deprecated > .title::after,
 span.deprecated::after {
   content: "Deprecated";
-  background: var(--deprecated-color);
+  background: var(--deprecated-background-color);
```

Comment:
Good spot. This will fix inline deprecations that use the old format (for example in [this clustering page](https://neo4j.com/docs/operations-manual/5/clustering/clustering-advanced/multi-data-center-routing/#mdc-programmatically-specify-rules)).

These instances should also be converted to use `label:deprecated[]`

Reply:
Unfortunately this hasn't fixed the problem with the old format, but I will investigate it